src/logicblocks/event/processing/broker/sources/factory/store.py

```diff
@@ -30,14 +30,37 @@ def construct_event_stream(
     return EventStream(adapter, identifier)
 
 
+type AdaptorConstructor[I: EventSourceIdentifier] = Callable[
```

Comment:
`Adapter`

Reply:
I think this should be an `EventSourceConstructor` - it doesn't construct an `Adapter`, it uses one to construct an `EventSource`.

Reply:
Fixed